go/runtime/txpool/txpool.go

```diff
@@ -126,6 +126,9 @@ type TransactionPool interface {
 
 	// PendingCheckSize returns the number of transactions currently pending to be checked.
 	PendingCheckSize() int
+
+	// GetPendingTxs returns the currently pending transactions.
```

Comment:
I think it would be more clear to just say that it returns all of transactions currently queued in the transaction pool. And maybe call the method `GetTxs()`.